connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExampleConnectIntegrationTest.java

```diff
@@ -157,7 +157,7 @@ public void testSinkConnector() throws Exception {
 
         // consume all records from the source topic or fail, to ensure that they were correctly produced.
```

Comment:
For me `consume all records` comments are different from `consumeAtLeast()` what you are intending for?

Maybe remove `all` ? 
```suggestion
        // consume records from the source topic or fail, to ensure that they were correctly produced.
```

Reply:
Ah, good catch! This comment has been copy+pasted across a large number of files:

- `ExampleConnectIntegrationTest`
- `ErrorHandlingIntegrationTest`
- `TransformationIntegrationTest`
- `RebalanceSourceConnectorsIntegrationTest`

In some cases, it's accurate. For example, at this part, we've just manually produced 2000 records into the topic, and we're now re-reading all of those records back to make sure that they were produced successfully.

In other cases, like line 218 in this file, the word "all" is inaccurate since we're consuming from a topic that's being actively written to by a running source connector, which has probably written more than the expected number of records by this point. I think removing "all" from the comment in this case would make it a little ambiguous as to what would cause the method invocation to fail. What do you think about "consume all expected records..." instead of "consume all records"?

Reply:
@C0urante 
Couldn't agree more!

Reply:
👍 done.